CONTRIBUTING.md

```diff
@@ -372,6 +372,9 @@ have the return type be either `None` or `Optional[bool]`. If you
 are not sure whether exceptions are suppressed or not or if the
 context manager is meant to be subclassed, pick `Optional[bool]`.
 See https://github.com/python/mypy/issues/7214 for more details.
+`__enter__` methods and other methods that return instances of the
```

Comment:
Maybe a blank line before this, because it's distinct from the `__exit__` stuff? It's not just a part of the `__exit__` explanation, because it also applies to other methods returning `self`.